lib/compat/wordpress-5.8/index.php

```diff
@@ -103,3 +103,64 @@ function build_query_vars_from_query_block( $block, $page ) {
 		return $query;
 	}
 }
+
+if ( ! function_exists( 'gutenberg_register_legacy_query_loop_block' ) ) {
```

Comment:
@desrosj: This is the wrong condition:

```diff
-if ( ! function_exists( 'gutenberg_register_legacy_query_loop_block' ) ) {
+if ( ! function_exists( 'gutenberg_render_legacy_query_loop_block' ) ) {
```

Reply:
Good spot. Sorry about that one! I'll fix it in a PR now.

Reply:
See #33513.